rust/tools/authorization-logic/src/souffle/universe_translation.rs

```diff
@@ -65,12 +65,6 @@ fn push_option_to_vec<T>(vec: &mut Vec<T>, elt: Option<T>) {
     }
 }
 
-fn push_if_constant(hash_set: &mut HashSet<String>, name: String) {
```

Comment:
This seems like a legitimately dead fn, @aferr if you'd like to keep it let me know and we can tag it.

Reply:
I think we should just delete this one, then.